functions/__fzf_search_current_dir.fish

```diff
@@ -3,7 +3,7 @@ function __fzf_search_current_dir --description "Search the current directory. R
     # See similar comment in __fzf_search_shell_variables.fish.
     set --local --export SHELL (command --search fish)
 
-    set fd_arguments --hidden --color=always --exclude=.git
+    set fd_opts --color=always $fzf_fd_opts
```

Comment:
renamed the variable because we are using opts everywhere else, e.g. FZF_DEFAULT_OPTS